### PR TITLE
Top level project information files

### PR DESCRIPTION
**Pull request by mahesh-hegde**
AUTHORS, CONTRIBUTING and README.

Not sure how exactly are merges resolved. I should have taken graph theory more seriously.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## jni_gen
+
+This project intends to provide
+
+* A code generator tools which generates Dart wrappers to Java libraries using FFI.
```
Comment (devoncarew) [inline README.md:5]: `* Code generation tools which generate ...`?

**Comment by devoncarew** [comment]
Hi - please keep a copy if the LICENSE file in `jni_gen/` (we should copy it to the root dir, not move it).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## jni_gen
```
Comment (dcharkes) [inline README.md:1]: Please add a build badge here as well.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+This project intends to provide
```
Comment (dcharkes) [inline README.md:3]: Maybe it would be cool to provide a table as in https://github.com/google/protobuf.dart.

**Review by dcharkes — COMMENTED** [review]
> Hi - as part of this change, we'll also want to delete the CONTRIBUTING.md file,

check

> move the authors file to the top level of the repo,

check

> and copy the license file to the top level

check

> (in addition to creating a top-level readme file).

check, and I left some comments on this one.

> 
> Here's an example of a top-level readme file for a monorepo: https://github.com/google/protobuf.dart. It has a brief description of the repo and uses a markdown table to direct people to the individual contained packages.

_Originally posted by @devoncarew in https://github.com/dart-lang/jni_gen/issues/2#issuecomment-1172585184_

File: README.md (hunk 2 above)
Comment (mahesh-hegde) [reply README.md:1]: Should I delete the one in the jni_gen directory?

Comment (dcharkes) [reply README.md:1]: Yes lets do that, the protobuf repo also doesn't have a build badge for the individual packages.

Comment (mahesh-hegde) [reply README.md:1]: Because the workflow runs on whole repo anyway?

Comment (dcharkes) [reply README.md:1]: Exactly https://github.com/google/protobuf.dart/blob/master/.github/workflows/dart.yml

**Review by dcharkes — APPROVED** [review]
lgtm!

**Comment by dcharkes** [comment]
> Not sure how exactly are merges resolved. I should have taken graph theory more seriously.

We squash and merge every PR. So it will just be the diff between the files currently on the last commit on the main branch and the final files in the PR. You can see the diff in "Files Changed" if you want to check.

**Pull request merged**